src/constants/common.ts

```diff
@@ -0,0 +1 @@
+export const BASE_URL = 'https://sp-taskify-api.vercel.app/2-5/';
```

Comment:
nit

`as const` 라고 하면 의도치 않은 값 변경을 방지할 수 있어요!

```suggestion
export const BASE_URL = 'https://sp-taskify-api.vercel.app/2-5/' as const;
```

Reply:
와우 리터럴 타입으로 하면 고정값으로 계속 사용할 수 있겠네요

Reply:
상수의 상수 같은건가...최고상수👍